### ISSUE TITLE
Clashes with `try`

### ISSUE THREAD

**Issue by DrHyde** [oodler577/p5-Try-ALRM#1]
It might be useful, as a way of working around the name-clash with try/catch, to be able to export your `try` function with a different name, such as `until_ALRM`.

**Comment by oodler577** [comment]
Hello! Hi thank you, I am not against renaming it. I am working on an updated version for release soon that'll contain a `retry` and `finally`. I speculate on this clash and defer the `try` in `Try::Tiny` in the POD. Anyway, will keep this open and think more about what the proper solution is going to be - even if I end up not exporting anything by default. I think this is where the real issue is, because it should be fine if someone did something like,

```
use Try::Tiny;
use Try::ALRM qw//;

..

Try::ALRM::try {

}
...
```

Thanks for the feedback. Feel free to add any more thoughts.

**Comment by oodler577** [comment]
I'd also like to point out that there is another module that exports `try` and `catch` (for that matter) - https://metacpan.org/pod/Nice::Try

**Comment by DrHyde** [comment]
and about a million others all subtlely incompatible!

**Comment by oodler577** [comment]
I've been thinking this over. `Try::ALRM::try` is really a reduced case of `retry` where `tries => 1`, so I think that `try_once { ...` might work; plus I get to delete lines of code, _POD_, and have just a single implementation in `retry`. Sounds like a win-win-win. Thanks again for your interest in resolving what could have been an impedement to adoption. I'll try to get that worked out today or tomorrow.

**Comment by oodler577** [comment]
Resolved with version 0.6, 7c06ac9ea07cf17b009d26333dd351d8d1d57040
